docs/tracking/google-tag-manager.md

```diff
@@ -1,17 +1,22 @@
 ---
-title: Google Tag Manager (GTM)
-description: Learn how to integrate Featurevisor SDK with Google Tag Manager
+title: Data Capturing via Google Tag Manager (GTM)
+description: Learn how to integrate Featurevisor SDK with Google Tag Manager and enable your experimentation reporting through custom dataLayer events
 ---
+## Intro:
```

Comment:
we can remove this part, in favour of our first paragraph ending with `{% .lead %}`, which will make the first paragraph appear in a bit larger text.

example: https://featurevisor.com/docs/use-cases/testing-in-production/